README.md

```diff
@@ -24,6 +24,8 @@ Role Variables
 * `podman_user_name` - name of the user running rootless podman. Default: osbs-podman.
 * `podman_user_uid` - uid of the user running rootless podman. Default: 2022.
 * `podman_user_ssh_public_key` - public key for podman user ssh and podman-remote access
+* `podman_pruning_every_minute` - minutes interval for podman pruning job
```

Comment:
the description is clearer than the variable name. I suggest having `podman_pruning_interval_minutes`, because `every_minute` is misleading (to me at least)

Reply:
+1, the current name sounds more like a boolean variable (but with no clear meaning for `false`)

Reply:
changed